scripts/run_experiment.sh

```diff
@@ -28,6 +28,11 @@ SED=$(sed -n 9p $run_path/submit.log)
 arrIN=(${SED//;/ })
 COLLECT_ID=$(echo ${arrIN[2]})
 echo "Collect_ID=$COLLECT_ID"
+if [[ $COLLECT_ID == "Batch" ]]
+then
+    cp $run_path/submit.log $HOME
```

Comment:
Kopiuj do folderu log w katalogu run-a

```suggestion
    mkdir -p ${BASE_DIR}/log/n_${N}/
    cp $run_path/submit.log ${BASE_DIR}/log/n_${N}/
```